lib/promoCard/PromoCard.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+
+class PromoCard extends StatefulWidget {
+  final Function() playHome;
```

Comment:
please use proper name like onButtonClickCallback

Reply:
changed function name to onButtonClickCallback